main/telemetry.js

```diff
@@ -25,41 +23,36 @@ const writeClient = client.getWriteApi(
 )
 
 setInterval(() => {
-  writeClient.flush().catch(err => {
-    // Ignore unactionable InfluxDB errors
-    // eslint-disable-next-line max-len
-    const reg = /HttpError|getAddrInfo|RequestTimedOutError|ECONNRESET|CERT_NOT_YET_VALID/i
-    if (!reg.test(String(err))) {
-      Sentry.captureException(err)
-    }
-  })
-}, 5000).unref()
```

Comment:
Would you mind shuffling the new code around to let `git diff` understand that we are not making any changes here?

Reply:
ok, will try!